### DISCUSSION TITLE
적용 안되는 모드 몇가지 있는 것 같아 말씀드립니다.

### DISCUSSION THREAD

**Discussion by WoojinSim** [RimWorldKorea/RMK#8] [일반]
`753498552` `Hospitality`
`2952321484` `Hospitality: Storefront`
`3014885065` `Hospitality: Vending machines`
`1127530465` `Dubs Rimatomics`
`728315620` `RT Solar Flare Shield`
`3014906877` `Vanilla Vehicles Expanded`
`3014906877` `Vanilla Vehicles Expanded - Tire 3`
`2424381146` `EvolvedOrgansRedux` (일부)

**Comment by RimWorldKorea** [comment]
#9
